test/testunusedvar.cpp

```diff
@@ -4842,14 +4842,20 @@ class TestUnusedVar : public TestFixture {
 
         // 2 translation units
         functionVariableUsage(
-            "#file \"file.h\"\n"
+            "#file \"file1.h\"\n"
             "class D {\n"
             "public:\n"
-            "   D() {};\n"
+            "   D();\n"
             "};\n"
             "#endfile\n"
-            "#file \"file.c\"\n"
-            "#include \"file.h\"\n"
+
+            "#file \"file1.c\"\n"
```

Comment:
now you have combined file1.c and file2.c, I am not sure what you want to do here but the code will be put in the same token list and that is not realistic. Imagine that you analyse a project. file1.c is analysed now and file2.c is analysed 10 minutes later. those are totally separated.